### PR TITLE
Blacksmitihng recipes QoL

### PR DESCRIPTION
**Pull request by Wirpy**
## About The Pull Request

QoL fixing for blacksmithing recipes. I have removed the steel, silver, iron etc in the names of certain recipes and just made them be the thing they are. Like steel sword -> sword since it's already implied that you are making a steel sword cause you are a using a bloody steel bar. 

This makes it less painful to navigate the recipe menu as well as now you can shrimply press thru the letter for the item you are looking for, I want to make sword I press s and isntead of everything named steel appearing now sword apepars you get the gist. 

Its great QoL.

## Why It's Good For The Game

Better less painful experience smithing + creates a cohesive naming convention for the recipes since they were all named in widly different ways. Looks cleaner and more cohesive now frfr.

### CODE REVIEW

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm [hunk 1]
```diff
@@ -3,61 +3,61 @@
...
 /datum/anvil_recipe/armor/studded
-	name = "studded leather armor (+leather armor)"
+	name = "Studded leather armor (+leather armor)"
 	req_bar = /obj/item/ingot/iron
 	additional_items = list(/obj/item/clothing/suit/roguetown/armor/leather)
 	created_item = /obj/item/clothing/suit/roguetown/armor/leather/studded
 
 /datum/anvil_recipe/armor/helmetgoblin
-	name = "iron goblin helmet (+iron)"
+	name = "Gblin helmet (+iron)"
```
Comment (FlickTee) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm:53]: minor typo

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm [hunk 2]
```diff
@@ -66,171 +66,171 @@
...
 
 /datum/anvil_recipe/armor/plateboot
-	name = "plated boots"
+	name = "Plated boots"
 	req_bar = /obj/item/ingot/steel
 	created_item = /obj/item/clothing/shoes/roguetown/boots/armor
 
 /datum/anvil_recipe/armor/platemask/steel
-	name = "steel mask"
+	name = "Mask"
```
Comment (FlickTee) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm:207]: two of these are just "mask" is that intentional? maybe mask (steel) ?

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm (hunk 1 above)
Comment (CVSthePharmacy) [reply code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm:53]: mind goblin helmet

**Comment by ChungusGamer666** [comment]
looks good, but conflicting

**Comment by ChungusGamer666** [comment]
not compiling, i tried to resolve conflicts but it didnt work due to shitty github webedit funnies
please fix

**Comment by Real-MAGNUM** [comment]
another reminder to please fix.

**Comment by Wirpy** [comment]
> not compiling, i tried to resolve conflicts but it didnt work due to shitty github webedit funnies please fix
Its allegedly fixed now. Just take a look to see if I missed any duplicate recipes otherwise g2g. I think I got all of the duplicate recipes that appeard after funny merge. 

> another reminder to please fix.
I was in school :(

**Comment by JTGSZ** [comment]
This removes decorating a already made sword, but they want to merge it so whatev lol

**Pull request merged**